frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
```

Comment:
I think we should use the company favicon instead of the react one

Reply:
where can I get one?

Reply:
whatever is better than the react one; for example:
- from Landing, https://sourced.tech/img/logos/favicon.png, or
- from Gitbase-web, https://gitbase-playground-staging.srcd.run/favicon.ico

Reply:
I believe we should use a new purple one like this one: https://avatars1.githubusercontent.com/u/15128793?s=32&v=4
Let me try to ask for it from the design team...

Reply:
if it's 32x32 I'd just use it here, and change whenever they release it if it has not been yet released.